clients/fides-js/src/fides.ts

```diff
@@ -83,7 +83,13 @@ const init = async (config: FidesConfig) => {
     dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
     dispatchFidesEvent("FidesUpdated", cookie, config.options.debug);
   }
-  const updatedFides = await initialize({ ...config, cookie, renderOverlay });
+  const experience = initialFides?.experience ?? config.experience;
```

Comment:
this is actually kind of a consequential change—basically what was originally happening was...

1. `getInitialFides` returns an object which may or may not have an updated experience based on cookie values. we'll call the updated experience `updatedExperience`
2. `_Fides`'s `experience` object is updated to `updatedExperience`
3. `initialize` is called with **the original experience object**

I believe that's a bug and we actually want to pass in the `updatedExperience` to `initialize`. Luckily, I think this only affects TCF since the notice overlay already builds its preferences off of cookies, not the experience object. But the TCF overlay builds it off of the experience object.